misc/distributed_search/dist_search_setup.md

```diff
@@ -0,0 +1,55 @@
+Distributed AlphaDIA search on HPCL
```

Comment:
Can this go to the documentation?
We have the `Methods` section here: https://alphadia.readthedocs.io/en/latest/methods.html